LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ProcessLogger.cs

```diff
@@ -18,12 +18,14 @@ internal sealed class ProcessLogger : IDisposable
 
         internal ProcessLogger(LoRaOperationTimeWatcher timeWatcher)
         {
+            this.LogLevel = LogLevel.Information;
```

Comment:
This is good so we are sure that this variable is set twice :-)

Reply:
I think it was messed up by the merge/rebase, since we both added this change.

Reply:
This code was meant to be quantum resistant, hence the double assignation.

Reply:
(or Francisco is right, you can pick your favorite explanation ;) )